UnityAudioFixer/UnityAudioFixer.cs

```diff
@@ -17,7 +17,7 @@ namespace QModManager
     public static class UnityAudioFixer
     {
         internal static string UnityAudioFixerPath => Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
-        internal static string GameRootPath => Path.Combine(UnityAudioFixerPath, "../../..");
+        internal static string GameRootPath => Environment.CurrentDirectory;
         internal static string DataPath => Directory.GetDirectories(GameRootPath, "*_Data", SearchOption.TopDirectoryOnly).SingleOrDefault();
```

Comment:
```suggestion
        internal static string DataPath => Directory.GetDirectories(Paths.GameRootPath, "*_Data", SearchOption.TopDirectoryOnly).SingleOrDefault();
```

Reply:
Not 100% sure if this suggestion will have any adverse affects, so do your due diligence :)

Otherwise, PR looks fine.